web/src/main/java/ua/com/vertex/controllers/exceptionHandling/GlobalExceptionHandler.java

```diff
@@ -50,6 +63,19 @@ public String handleCannotGetJdbcConnectionException(Exception e, Model model) {
         return ERROR;
     }
 
+    @ExceptionHandler(MultipartValidationException.class)
+    public String handleMultipartValidationException(MultipartValidationException e, Model model) {
+        if (FILE_SIZE_EXCEEDED.equals(e.getMessage())) {
```

Comment:
can you please add readable message to exception and after that just add e.getMessage() as error message to model? Without if-else blocks

Reply:
Done